src/pages/apps/[id]/environments/[env-id]/deployments/runtime-logs/RuntimeLogs.tsx

```diff
@@ -57,14 +69,14 @@ const RuntimeLogs: React.FC = () => {
       }
       maxWidth="max-w-none"
     >
-      <div className="p-4 pt-0 max-h-screen" style={{ overflowY: "auto" }}>
-        {loading && (
+      <div className="p-4 pt-0">
+        {loading && page === 0 && (
```

Comment:
This allows appending logs and do not display a loading spinner when `Load more logs` button is clicked.